src/grt/src/fastroute/src/maze3D.cpp

```diff
@@ -895,8 +895,8 @@ void FastRouteCore::mazeRouteMSMDOrder3D(int expand,
 
     int enlarge = expand;
     const int num_terminals = sttrees_[netID].num_terminals;
-    const auto& treeedges = sttrees_[netID].edges;
-    const auto& treenodes = sttrees_[netID].nodes;
+    auto& treeedges = sttrees_[netID].edges;
+    auto& treenodes = sttrees_[netID].nodes;
```

Comment:
Why they're losing const? Here and below.

Reply:
Because those variables are then modified in the function

Reply:
I didn't notice some function calls that use these variables were marked as non-const.